dev-ml/camomile/camomile-0.8.5-r2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit findlib eutils
+
+DESCRIPTION="Camomile is a comprehensive Unicode library for ocaml"
+HOMEPAGE="https://github.com/yoriyuki/Camomile/wiki"
+SRC_URI="https://github.com/yoriyuki/Camomile/releases/download/rel-${PV}/${P}.tar.bz2"
+
+LICENSE="LGPL-2"
+SLOT="0/${PV}"
+KEYWORDS="amd64 ppc x86"
```

Comment:
I should have spotted this the first time: committing straight to stable is something to do with caution, but actually, if this is a build fix (i.e. it either builds, or it doesn't - in which case it'll keep trying to build when you update), we don't have to revbump at all. 

We just add the fix to the existing ebuild.

Reply:
> committing straight to stable is something to do with caution

Dangit, I knew that, and remembered on an unrelated PR I did today. In fact I wonder why repoman didn't yell at me about this.

Anyway, fixed this and the other things with ocaml-gettext.

Folding into the existing ebuild rather than bumping -rN would probably work for both, since they both depend on dev-ml/ocaml=, they'd already get a forced rebuild upon ocaml upgrade. But I don't know if the EAPI bump would necessitate it.  (I have a personal preference for almost-always bumping because I've been bitten when there was no bump and there should have been, and that is the worst kind of time-wasting ghost to chase... </soapbox>.)